salt/etc-hosts/hosts.jinja

```diff
@@ -14,6 +14,9 @@
 {%- for admin_id, ifaces in admins.items() %}
 {{ salt.caasp_net.get_primary_ip(host=admin_id, ifaces=ifaces) }} {{ salt.caasp_net.get_nodename(host=admin_id) }} {{ salt.caasp_net.get_nodename(host=admin_id) }}.{{ pillar['internal_infra_domain'] }} {{ admin_id }} {{ admin_id }}.{{ pillar['internal_infra_domain'] }}
 {%- endfor %}
+{% if "admin" in salt['grains.get']('roles', []) %}
+127.0.0.1 ldap.{{ pillar['internal_infra_domain'] }}
```

Comment:
This duplicates line 9 above - and should be folded into that if/else branch.

Reply:
updated